### PR TITLE
Switched to travis-ci and test multiple django versions.

### PR DESCRIPTION
**Pull request by Surgo**
Travis-CI results of my fork: [See](https://travis-ci.org/uncovertruth/django-sharding)

TODO:

* [x] Change badge in README.md

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=['Django>=1.8', 'dj-database-url==0.3.0'],
-    tests_require=['psycopg2==2.6.1', 'mysqlclient==1.3.7', 'mock==1.0.1', 'django_nose==1.4.2', 'coverage==4.0', 'coveralls==1.0', 'tox==2.1.1'],
```
Comment (Surgo) [inline setup.py]: coverage and coveralls are only needed in the ci test.

File: requirements/common.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+django>=1.8
```
Comment (Surgo) [inline requirements/common.txt]: duplication preventing in tox's django.

**Comment by JBKahn** [comment]
Nice work so far, I jus enabled travis for the repo so if it picks it up then on your next push it should be there alongside circle.

**Comment by JBKahn** [comment]
Let me know if you'd like something from me. I can try to help, but you seem to be off in a good direction.

**Comment by 4word** [comment]
How is this looking? I am all for moving over to travis, but watching travis fail in other pull requests because its not configured correctly in master yet sucks!

**Comment by Surgo** [comment]
sorry, travis-ci work after `push -f` :(

**Comment by Surgo** [comment]
ummmm... I could not understand why failed this test.

https://travis-ci.org/JBKahn/django-sharding/jobs/128229439#L250

```
        self.assertTrue(mock_migrate_command.call_args[1].get('fake'))  // passed
        self.assertTrue(mock_migrate_command.call_args[1].get('fake_initial'))  // passed
        self.assertFalse(mock_migrate_command.call_args[1].get('interactive'))  // failed
```

and that are not repeatability...

**Comment by JBKahn** [comment]
Hmm I'll take a look tonight

**Comment by JBKahn** [comment]
been super busy, I'll take a look right now.

**Comment by JBKahn** [comment]
I think we'll have to change the test, it's a bug in Django with arg_parsing and depends on the return value of dict.items(). I'm going to open a PR to fix it in Django. For now, have it raise a skip test.

**Comment by JBKahn** [comment]
Here's the bug report I filed: https://code.djangoproject.com/ticket/26597#ticket

**Comment by Surgo** [comment]
oh... thank you! skip that.

**Pull request merged**